### PR TITLE
Fix website.

### PR DESCRIPTION
**Pull request by Alexays**
- Correct indenting files.
- Fix overflow Ribbon.
- Center box (yes current it's not).
- Fix colors first frame canvas x)
- Correct background canvas which should be fafafa and not ffffff.
edit: Transparent background now.
Add fix for card, add posibility to have a card with big content (ex: voters list).

### CODE REVIEW

File: server/index.html [hunk 1]
```diff
@@ -1,65 +1,54 @@
...
+        <link rel="apple-touch-icon" sizes="180x180" href="static/assets/icons/apple-icon-180x180.png">
+        <link rel="icon" type="image/png" sizes="192x192"  href="static/assets/icons/android-icon-192x192.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="static/assets/icons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="96x96" href="static/assets/icons/favicon-96x96.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="static/assets/icons/favicon-16x16.png">
+        <link rel="manifest" href="manifest.json">
+        <meta name="msapplication-TileColor" content="#ffffff">
+        <meta name="msapplication-TileImage" content="static/assets/icons/ms-icon-144x144.png">
+        <meta name="theme-color" content="#ffffff">
+        <meta property="og:url"                content="http://chaosthebot.com/" />
```
Comment (aslafy-z) [inline server/index.html]: Could be the right time to harmonize the spacings here

Comment (Alexays) [reply server/index.html]: Yes I will do it ;)

File: server/static/js/chaos.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ var ctx = canvas.getContext("2d");
 canvas.width = 400;
 canvas.height = 200;
 
-ctx.fillStyle = "#000000";
+ctx.fillStyle = "#fffafa";
```
Comment (phil-r) [inline server/static/js/chaos.js]: is it intended that it's not `#fafafa` ?

File: server/static/js/chaos.js [hunk 3]
```diff
@@ -32,82 +32,84 @@ canvas.onmousemove = function(e) {
...
-    for (x = 0; x < pix_location.length; x++) {
-      var xPos = pix_location[x][0] + 0.1*(char_location[x][0]-pix_location[x][0]);
-      var yPos = pix_location[x][1] + 0.1*(char_location[x][1]-pix_location[x][1]);
-      if (lastX > 0 && lastY > 0) {
-          dx = pix_location[x][0]-lastX;
-          dy = pix_location[x][1]-lastY;
-          if (Math.sqrt(dx*dx+dy*dy) < 10) {
-            yPos = Math.random() > 0.5 ? 0 : canvas.height;
-          }
+  ctx.fillStyle = "#fafafa";
```
Comment (phil-r) [inline server/static/js/chaos.js]: maybe you can use a variable for `"#fafafa"` ?

File: server/static/js/chaos.js (hunk 2 above)
Comment (Alexays) [reply server/static/js/chaos.js]: Yeah thx.

File: server/static/js/chaos.js (hunk 3 above)
Comment (Alexays) [reply server/static/js/chaos.js]: It's done :)

**Comment by phil-r** [comment]
Maybe I'm missing something, but why we are not using transparent background for canvas?
Example: https://jsfiddle.net/Phil/1e8Lc757/

**Pull request merged**